app/src/app/list/list.component.html

```diff
@@ -1,9 +1,15 @@
 <app-server-error [error]="error$ | async"></app-server-error>
-<div class="poll-list" *ngIf="(publicList$ | async)?.length">
-  <h3>Polls you participate in</h3>
-  <app-polls-table [polls]="publicList$ | async"></app-polls-table>
-</div>
-<div class="poll-list" *ngIf="(ownList$ | async)?.length">
-  <h3>Polls you have created</h3>
-  <app-polls-table [polls]="ownList$ | async"></app-polls-table>
-</div>
+<mat-tab-group mat-align-tabs="center" id="tab-polls">
+  <mat-tab  label="Participate" class="poll-list" *ngIf="(publicList$ | async)?.length">
+    <div class="poll-list">
```

Comment:
It's very strange to have to element, on inside the other, both with the same class.